db/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@filecoin-station/spark-stats-db",
```

Comment:
Hmm, this package does not maintain the database (e.g. the migration scripts), only the client connection factories, right? The name `db` seems a bit misleading then. 

I'd like to propose two options:
- Rename `migrations` package to `db` and merge it with the code introduced in this PR.
- Rename the newly introduced package to something else, e.g. `@filecoin-station/spark-stats-db-clients`

Thoughts?

Reply:
hm, but what about this?

```
import { migrate } from '@filecoin-station/spark-stats-db-clients'
```